sympy/physics/mechanics/tests/test_body.py

```diff
@@ -273,3 +274,53 @@ def test_apply_loads_on_multi_degree_freedom_holonomic_system():
     assert P.loads == [(P.masscenter, P.mass*g*W.y), (P.frame, (T + kT*q2)*W.z)]
     assert b.loads == [(b.masscenter, b.mass*g*W.y), (b.frame, -kT*q2*W.z)]
     assert W.loads == [(W.masscenter, (c*q1.diff() + k*q1)*W.x)]
+
+def test_orient_axis():
+    A = Body('A')
+    B = Body('B')
+    N = ReferenceFrame('N')
+    A.orient_axis(N, N.x + N.y, 10)
+    B.orient_axis(A, A.x + A.y, 10)
+    dcm = Matrix([[cos(10)/2 + S.Half, S.Half - cos(10)/2, -sqrt(2)*sin(10)/2], [S.Half - cos(10)/2,
+    cos(10)/2 + S.Half, sqrt(2)*sin(10)/2], [sqrt(2)*sin(10)/2, -sqrt(2)*sin(10)/2, cos(10)]])
```

Comment:
Use tabs for proper indentation , else everything seems fine.

Reply:
Check other matrices indentation.

Reply:
https://github.com/sympy/sympy/blob/2c83657ff1c62fc2761b639469fdac7f7561a72a/sympy/physics/vector/tests/test_frame.py#L470
I saw matrices being written in a single line here so I used this format. Let me know if this is ok.

Reply:
https://github.com/sympy/sympy/blob/25a3b8ec67a2c0ae99d0be2e9f01d294e207e1ad/sympy/physics/mechanics/tests/test_joint.py#L82-L84

Make it like this.

Reply:
On the second though it seems fine.